tools/analysis_tools/bio3d_pca.xml

```diff
@@ -0,0 +1,157 @@
+<tool id="bio3d_PCA_analysis" name="PCA" version="0.1.2">
+    <description>Principle component analysis using Bio3D</description>
+    <requirements>
+          <requirement type="package" version="2.3_3">r-bio3d</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command interpreter="Rscript">
+<![CDATA[ 
+    bio3d_pca.R 
+        $dcdin 
```

Comment:
Please always single-quote all data-parameters and all text parameters. That's important for security.